lib/elixir/src/elixir_def.erl

```diff
@@ -99,6 +99,10 @@ store_definition(Kind, CheckClauses, Call, Body, Pos) ->
   %% extract meta information like file and context.
   {_, Meta, _} = Call,
   DoCheckClauses = (not lists:keymember(context, 1, Meta)) andalso (CheckClauses),
+  Generated = case lists:keyfind(generated, 1, Meta) of
```

Comment:
This is related to https://github.com/elixir-lang/elixir/blob/master/lib/elixir/test/elixir/module_test.exs#L291